.github/workflows/ci.yml

```diff
@@ -294,20 +294,25 @@ jobs:
           python test/check_py27_compat.py
 
 
-  # cpp_check:
-  #   runs-on: 'ubuntu-latest'
-  #   steps:
+  cpp_check:
+    name: Cppcheck static analysis
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
 
-  #   # - template: azure-templates/cppcheck.yml
-  #   # - template: azure-templates/python.yml
+    # - template: azure-templates/cppcheck.yml
+    # - template: azure-templates/python.yml
+    #
```

Comment:
These lines can be deleted now -- they were the recipes for installing cppcheck and python.